setup.py

```diff
@@ -14,7 +14,7 @@
     "PyYAML==3.12",
 ]
 
-VERSION = "4.3.1"
+VERSION = "4.3.0"
```

Comment:
`master` is currently on 4.3.4
https://github.com/genomicsengland/GelReportModels/blob/b66324603773f6b42121b73a0ad5ab5236145e67/setup.py#L20

Do we want to make this 4.4.0 or something like that ?

Reply:
aagggh I changed it in CHANGES.md and forgot about setup.py...
I thought about 4.3.5

Reply:
done